### DISCUSSION TITLE
JPGs

### DISCUSSION THREAD

**Discussion by mystichussar** [mariush444/raster2osmand#1] [General]
Hi there!

Trying to convert jpg maps from http://igrek.amzp.pl/mapindex.php?cat=TK25&listtype=standard&listsort=sortoption1 but with no success. Can you help with that?

BR

**Comment by mariush444** [comment]
The raster maps have to be geotagged (calibrated) like geotiff, geopdf or jpg with ozi calibrated file.
The ones from amzp.pl are just jpg without any georefference.
My script can't help you in this case.
